docs/source/conf.py

```diff
@@ -182,6 +182,12 @@
 # Output file base name for HTML help builder.
 htmlhelp_basename = 'mgrsdoc'
 
+# Substitutions
+rst_epilog = """
+.. |BLQGIS| replace:: *QGIS for OpenGeo Suite*
```

Comment:
For now, it's still Boundless Desktop. The substitution was created thinking in the change.